python/mergeTreeClustering.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'XML Image Data Reader'
+isabelvti = XMLImageDataReader(registrationName='isabel.vti', FileName=['isabel.vti'])
+isabelvti.PointArrayStatus = ['velocityMag_02', 'velocityMag_03', 'velocityMag_04', 'velocityMag_05', 'velocityMag_30', 'velocityMag_31', 'velocityMag_32', 'velocityMag_33', 'velocityMag_45', 'velocityMag_46', 'velocityMag_47', 'velocityMag_48']
+isabelvti.TimeArray = 'None'
+
+all_JT_group = []
+all_ST_group = []
+
+for scalarField in isabelvti.PointArrayStatus:
+  # create a new 'TTK Merge and Contour Tree (FTM)'
+  tTKMergeandContourTreeFTM1 = TTKMergeandContourTreeFTM(Input=isabelvti)
+  tTKMergeandContourTreeFTM1.ScalarField = ['POINTS', scalarField]
+  tTKMergeandContourTreeFTM1.InputOffsetField = ['POINTS', scalarField]
+  tTKMergeandContourTreeFTM1.TreeType = 'Join Tree'
+  
+  # create a new 'Group Datasets'
+  groupDatasets1 = GroupDatasets(Input=[tTKMergeandContourTreeFTM1, OutputPort(tTKMergeandContourTreeFTM1,1), OutputPort(tTKMergeandContourTreeFTM1,2)])
+  
+  all_JT_group.append(groupDatasets1)
+  
+  # create a new 'TTK Merge and Contour Tree (FTM)'
+  tTKMergeandContourTreeFTM2 = TTKMergeandContourTreeFTM(Input=isabelvti)
+  tTKMergeandContourTreeFTM2.ScalarField = ['POINTS', scalarField]
+  tTKMergeandContourTreeFTM2.InputOffsetField = ['POINTS', scalarField]
+  tTKMergeandContourTreeFTM2.TreeType = 'Split Tree'
+  
+  # create a new 'Group Datasets'
+  groupDatasets2 = GroupDatasets(Input=[tTKMergeandContourTreeFTM2, OutputPort(tTKMergeandContourTreeFTM2,1), OutputPort(tTKMergeandContourTreeFTM2,2)])
+  
+  all_ST_group.append(groupDatasets2)
+
+# create a new 'Group Datasets'
+mt_JT_all = GroupDatasets(Input=all_JT_group)
+
+# create a new 'Group Datasets'
+mT_all = GroupDatasets(Input=all_ST_group)
+
+# create a new 'TTK MergeTreeClustering'
+tTKMergeTreeClustering1 = TTKMergeTreeClustering(Input=mT_all,
+    OptionalInputclustering=mt_JT_all)
+tTKMergeTreeClustering1.ComputeBarycenter = 1
+tTKMergeTreeClustering1.NumberOfClusters = 3
+tTKMergeTreeClustering1.Deterministic = 1
+tTKMergeTreeClustering1.DimensionSpacing = 0.1
```

Comment:
Isn't it the default value? (if so, this line should be removed)

Reply:
No, it isn't the default value.